Makefile

```diff
@@ -176,6 +176,13 @@ out/minikube_$(DEB_VERSION).deb: out/minikube-linux-amd64
 	dpkg-deb --build out/minikube_$(DEB_VERSION)
 	rm -rf out/minikube_$(DEB_VERSION)
 
+out/minikube_$(DEB_VERSION).tar.gz: out/localkube out/minikube-linux-amd64 out/minikube-darwin-amd64 out/minikube-windows-amd64.exe
```

Comment:
Even if we don't include them now, if we don't release anything we can still defer the decision as we stub out what the full design looks like.